### PR TITLE
Print detailed error when error happens

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Error of the func `Stop` is never used though it happens. Print it when error happens.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Thanks for this pr :)

I can't see any situation in which it could hurt to have this information or it could pollute the logs, and it seems like it has the chance to help with debugging.

**Comment by cwdsuzhou** [comment]
> /lgtm
> 
> Thanks for this pr :)
> 
> I can't see any situation in which it could hurt to have this information or it could pollute the logs, and it seems like it has the chance to help with debugging.

Yeah, thanks

**Comment by cwdsuzhou** [comment]
ping @vishh

**Comment by cwdsuzhou** [comment]
ping @tallclair

**Comment by cwdsuzhou** [comment]
ping @vishh @tallclair @Random-Liu @dchen1107 @derekwaynecarr 

could anyone help approve this PR?

File: pkg/kubelet/util/pluginwatcher/plugin_watcher.go [hunk 1]
```diff
@@ -104,14 +104,18 @@ func (w *Watcher) Start() error {
 
 	// Traverse plugin dir and add filesystem watchers before starting the plugin processing goroutine.
 	if err := w.traversePluginDir(w.path); err != nil {
-		w.Stop()
+		if stopError := w.Stop(); stopError != nil {
+			klog.Error(stopError)
```
Comment (tallclair) [inline pkg/kubelet/util/pluginwatcher/plugin_watcher.go]: Please add some context to the error.
```suggestion
			klog.Errorf("Failed to stop watching plugins at %s: %v", w.path, stopError)
```

File: pkg/kubelet/util/pluginwatcher/plugin_watcher.go [hunk 2]
```diff
@@ -104,14 +104,18 @@ func (w *Watcher) Start() error {
...
+		}
 		return fmt.Errorf("failed to traverse plugin socket path %q, err: %v", w.path, err)
 	}
 
 	// Traverse deprecated plugin dir, if specified.
 	if len(w.deprecatedPath) != 0 {
 		if err := w.traversePluginDir(w.deprecatedPath); err != nil {
-			w.Stop()
+			if stopError := w.Stop(); stopError != nil {
+				klog.Error(stopError)
```
Comment (tallclair) [inline pkg/kubelet/util/pluginwatcher/plugin_watcher.go]: ditto

Comment (cwdsuzhou) [reply pkg/kubelet/util/pluginwatcher/plugin_watcher.go]: Done

File: pkg/kubelet/util/pluginwatcher/plugin_watcher.go (hunk 1 above)
Comment (cwdsuzhou) [reply pkg/kubelet/util/pluginwatcher/plugin_watcher.go]: Done

**Comment by cwdsuzhou** [comment]
ping @tallclair 

All have done. PAL again, thanks!

**Comment by cwdsuzhou** [comment]
priority/backlog

**Comment by cwdsuzhou** [comment]
ping @tallclair @derekwaynecarr PTAL

**Pull request closed without merging**